syncd/FlexCounter.cpp

```diff
@@ -995,10 +1080,41 @@ class AttrContext : public CounterContext<AttrType>
                 values.emplace_back(meta->attridname, sai_serialize_attr_value(*meta, attrs[i]));
             }
             countersTable.set(sai_serialize_object_id(vid), values, "");
+            if (switchType == "voq")
+            {
+               // Chassis based system
+               appDbCountersTable->set(sai_serialize_object_id(vid), values, "");
+            }
         }
     }
 };
 
+void FlexCounter::getCfgSwitchType(
+         _In_ std::string &switch_type)
```

Comment:
you are changing output variable, yo ushould use _ Out _, or better make function as return std::string instead of modyfying input parameter

Reply:
Done